src/alarms/notifications/App.js

```diff
@@ -58,21 +59,122 @@ class App extends Component {
       .then(response => response.json())
       .then(data => {
         this.setState({
-          isFetching: false,
-          total: data.count,
-          alarms: data.results,
+          isFetchingRasterAlarms: false,
+          total: this.state.total + data.count,
+          alarms: this.state.alarms.concat(
+            data.results.map(e => {
+              e.type = "RasterAlarm";
+              return e;
+            })
+          ),
+          page: page
+        });
+      });
+  }
+
+  loadTimeseriesAlarms(page, newProps) {
+    const { selectedOrganisation } = newProps || this.props;
+    const organisationId = selectedOrganisation.uuid;
+
+    fetch(
+      `/api/v3/timeseriesalarms/?page=${page}&organisation__unique_id=${organisationId}`,
+      {
+        credentials: "same-origin"
+      }
+    )
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          isFetchingTimeseriesAlarms: false,
+          total: this.state.total + data.count,
+          alarms: this.state.alarms.concat(
+            data.results.map(e => {
+              e.type = "TimeseriesAlarm";
+              return e;
+            })
+          ),
           page: page
         });
       });
   }
 
+  urlFromAlarm(alarm) {
```

Comment:
url van alrm is nu afhankelijk van het type alarm.
Dit is een terugkerend stukje logica voor volgende 3 functies en heb ik daarom maar in deze functie gezet.

Reply:
Heeft een Alarm niet al een URL veld vanuit de API? Zo niet dan zou het een method op het TimeseriesAlarm / RasterAlarm object in lizard-api-client moeten zijn.